back/spec/models/news_post_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe NewsPost, type: :model do
+  describe "Default factory" do
+    it "is valid" do
+      expect(build(:news_post)).to be_valid
```

Comment:
It's absolutely fine, just as an alternative, you could use `it { is_expected.not_to be_valid }` similar to back/engines/commercial/insights/spec/models/insights/category_spec.rb